backend/entities/Parcel.cs

```diff
@@ -105,7 +110,7 @@ public class Parcel : BaseEntity
         /// <summary>
         /// get/set - Potential future Parcel zoning information
         /// </summary>
-        public bool ZoningPotential { get; set; }
+        public string ZoningPotential { get; set; }
```

Comment:
Fix bug as this should be a string, not boolean.

Reply:
Thanks for fixing this.